src/test/java/org/folio/rest/impl/SynchronizationAPITests.java

```diff
@@ -160,6 +170,41 @@ public void declaredLostEventShouldBeCreatedAfterSynchronization() {
     checkSyncJobUpdatedByLoanEvent(syncJobId);
   }
 
+  @Test
+  public void agedToLostEventShouldBeDeletedAfterSynchronizationJobByUser() {
+    sendAgedToLostEvent(createItemAgedToLostEvent(USER_ID), SC_NO_CONTENT);
+
+    stubLoans(now().plusHours(1).toDate(), true, "Checked out");
+    stubAccountsWithEmptyResponse();
+    String syncJobId = createOpenSynchronizationJobByUser();
+
+    runSynchronization();
+
+    Awaitility.await()
+      .atMost(5, SECONDS)
+      .until(() -> waitFor(itemAgedToLostEventRepository.getByUserId(USER_ID)).size(), is(0));
+
+    checkSyncJobUpdatedByLoanEvent(syncJobId);
+  }
+
+  @Test
+  public void agedToLostEventsShouldBeDeletedAfterSynchronizationJobFull() {
```

Comment:
They should be deleted BEFORE synchronization starts

Reply:
fixed